napari/components/_interaction_box_constants.py

```diff
@@ -0,0 +1,18 @@
+class Box:
+    """Box: Constants associated with the vertices of the interaction box"""
+
+    # List of points to include for vertices
```

Comment:
Thanks for the explanatory comments - I think these were added from the last PR!

Reply:
Sure, I think I addressed most of your comments from the last PR.